test/register.test.js

```diff
@@ -0,0 +1,45 @@
+import quidditchProducts from '../src/data/quidditch.js';
+import order from '../src/data/order.js';
+import { findProduct, getLineTotal, getOrderTotal } from '../src/register.js';
+
+const test = QUnit.test;
+
+QUnit.module('Register');
+
+test('Find product by product code in quidditch.js', assert => {
+
+    const code = 'nimbus-2000';
+    const expected = {
+        code: 'nimbus-2000',
+        name: 'Nimbus 2000',
+        image: 'assets/nimbus-2000.png',
+        description: 'A speedy, nimble, top-of-the-line broomstick',
+        category: 'broomstick',
+        price: 5000.00,
+        cost: 500.00
+    };
+    
+    const foundProduct = findProduct(quidditchProducts, code);
+
+    assert.deepEqual(foundProduct, expected);
+});
+
+test('Calculate line item total from getLineTotal in register.js', (assert) => {
```

Comment:
sometimes you have the parenthesis around assert and sometimes you don't in your test files. You don't need them.